apps/dashboard/src/components/workflow-editor/steps/email/repeat-menu-description.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { Editor, useEditorState } from '@tiptap/react';
+import { AnimatePresence, motion } from 'motion/react';
+import { Lightbulb } from 'lucide-react';
+import { Separator } from '@/components/primitives/separator';
+
+export function RepeatMenuDescription({ editor }: { editor: Editor }) {
+  const [currentProperty, setCurrentProperty] = useState('\u00A0}}');
+  const state = useEditorState({
+    editor,
+    selector: (ctx) => {
+      return {
+        each: ctx.editor.getAttributes('repeat')?.each,
+        currentShowIfKey: ctx.editor.getAttributes('repeat')?.showIfKey || '',
+
+        isSectionActive: ctx.editor.isActive('section'),
+      };
+    },
+  });
+  const forNodeEachKey = state.each;
+
+  function isOnEmptyLine(editor: Editor, cursorPos: number) {
+    const currentLineContent = editor.state.doc
+      .textBetween(
+        Math.max(0, editor.state.doc.resolve(cursorPos).start()),
+        Math.min(editor.state.doc.content.size, editor.state.doc.resolve(cursorPos).end())
+      )
+      .trim();
+
+    return currentLineContent === '';
+  }
+
+  useEffect(() => {
+    const properties = ['\u00A0}}', '.foo }}', '.bar }}', '.attr }}'];
+    let currentIndex = 0;
+
+    const interval = setInterval(() => {
+      currentIndex = (currentIndex + 1) % properties.length;
+      setCurrentProperty(properties[currentIndex]);
+    }, 2000);
+
+    return () => clearInterval(interval);
+  }, []);
+
+  const shouldShow = isOnEmptyLine(editor, editor.state.selection.from);
+
+  return (
+    <AnimatePresence mode="wait">
+      {shouldShow && (
+        <motion.div
+          key="repeat-menu"
+          initial={{ opacity: 0, height: 0 }}
+          animate={{ opacity: 1, height: 'auto' }}
+          exit={{ opacity: 0, height: 0 }}
+          transition={{ duration: 0.25, ease: 'easeInOut' }}
+          className="mly-shadow-sm mly-text-gray-400 overflow-hidden text-xs"
+        >
+          <Separator className="mt-0.5" />
+          <div className="flex items-start gap-1 px-1 py-1.5">
+            <Lightbulb className="mt-0.5 size-3.5 stroke-[2] text-gray-400" />
+            <div>
+              <div>Access each 'repeat' key via</div>
```

Comment:
Ensure that the min-width of the bubble is enough to show this in a single line. You can also change copywriting to `Access 'repeat' key via `